src/components/common/Header.js

```diff
@@ -129,74 +90,116 @@ const useStyles = makeStyles(() => ({
+  { id: 'breadSubItem5', text: 'breadSubItem5' },
+  { id: 'breadSubItem6', text: 'breadSubItem6' },
+  { id: 'breadSubItem7', text: 'breadSubItem7' },
+  { id: 'composition2', text: 'Composition 2' },
+];
 
 const Header = () => {
   const classes = useStyles();
   const [anchorEl, setAnchorEl] = React.useState(null);
   const open = Boolean(anchorEl);
 
   const handleClick = (event) => {
-    setAnchorEl(anchorEl ? null : event.currentTarget);
+    if (event) {
+      setAnchorEl(event.currentTarget);
+    }
   };
 
-  const id = open ? 'simple-popper' : undefined;
-
-  const label = { inputProps: { 'aria-label': 'Switch demo' } };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+  const id = open ? 'composition-popper' : undefined;
 
   return (
-    <Box className={classes.root}>
-      <Box className={classes.leftSection}>
-        <Button
-          disableRipple
-          aria-describedby={id}
-          onClick={handleClick}
-        >
-          model12.py
-          <img src={DOWN} alt="down" />
-        </Button>
-
-        <Popper id={id} open={open} className={classes.dropDown} anchorEl={anchorEl} placement='bottom-start'>
-          <Box className={classes.dropDownHeader}>
-            <img src={LOGO} alt="logo" />
-            <Box>
-              <Typography component="strong">model12.py</Typography>
-              <Typography component="span">PsyNeuLink View</Typography>
-            </Box>
-          </Box>
-
-          <Box className={classes.card}>
-            <Typography>Background dotted grid</Typography>
-            <Switch {...label} defaultChecked />
-          </Box>
-
-          <Box className={classes.card}>
-            <Typography>About Psyneulink</Typography>
-             <IconButton disableRipple><img src={INFO} alt="info" /></IconButton>
-          </Box>
-        </Popper>
-        {/* <Typography>New visualization</Typography> */}
+    <>
+      <Dialog
+        id={id}
+        open={open}
+        hideBackdrop
+        PaperProps={{
+          sx: {
+            position: 'fixed',
+            top: 96,
+            left: 60,
+            width: 'calc(100VW - 24.25rem)',
+            maxWidth: 'calc(100VW - 24.25rem)',
+            height: 'calc(100Vh - 11rem)',
+            border: `2px solid ${dialogBorderColor}`,
+            background: headerBorderColor,
+            borderRadius: '0.75rem',
+            m: 0,
+          },
+        }}
+        aria-labelledby="composition-popper"
+      >
+        <Typography>Composition 2</Typography>
+      </Dialog>
+      <Box className={classes.root}>
+        <Box className={classes.leftSection}>
+          <img src={PSYLOGO} alt="new-logo" aria-describedby="logo" />
+
+          <CustomBreadcrumbsWithMenu breadcrumbs={breadcrumbs} />
+        </Box>
+        <Box className={classes.middleSection}>
+          <List className="headerSwitch" component="nav">
+            <ListItemButton disableRipple selected>
```

Comment:
imo you should not repeat yourself... this could be rewritten like this:
```
const listItems = [{label:'Build',soon: false}, {label:'Test', soon: true}, {label:'Composition', soon:false}]
listItems.map((item) => {
retrun (
  <ListItemButton disableRipple {item.soon ? disabled : onClick={handleClick}>
    <Typography>{item.label}</Typography>
    {item.soon && <Chip label="SOON" />}
  </ListItemButton>
)
})
```
instead of the inline array you could create a